src/main/java/seedu/address/storage/JsonSerializableArticleBook.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.storage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.ArticleBook;
+import seedu.address.model.ReadOnlyArticleBook;
+import seedu.address.model.article.Article;
+
+/**
+ * An immutable ArticleBook that cans be serialized into Json format
+ */
+public class JsonSerializableArticleBook {
+    public static final String MESSAGE_DUPLICATE_ARTICLE = "Articles list contains duplicate article(s).";
+
+    private final List<JsonAdaptedArticle> articles = new ArrayList<>();
+
+    /**
+     * Constructs a {@code JsonSerializableAddressBook} with the given persons.
```

Comment:
Should change `JsonSerializableAddressBook` to `JsonSerializableArticleBook`, `persons` to `articles`